sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/models/AzureCliToken.java

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.implementation.models;
+
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A wrapper class for deserializing a token payload returned from the Azure CLI.
+ */
+public class AzureCliToken implements JsonSerializable<AzureCliToken> {
+    private static final OffsetDateTime EPOCH = OffsetDateTime.of(1970, 1, 1, 0, 0, 0, 0, ZoneOffset.UTC);
```

Comment:
After this PR, should look into using `Instant.ofEpochSecond`

Reply:
Ah! I could not find this API, kept looking on all the `*DateTime` types. Thanks.